packages/utils/src/modules-sdk/internal-module-service-factory.ts

```diff
@@ -63,6 +63,48 @@ export function internalModuleServiceFactory<
       return keys.map((k) => data[k]).join("_")
     }
 
+    static applyDefaultOrdering(config: FindConfig<any>) {
+      if (config.order) {
+        return
+      }
+
+      const order: object[] = []
```

Comment:
We can make the type stricter, where the key is `keyof model`, and the value is "ASC" or "DESC"